apps/server/src/modules/files/service/files.service.spec.ts

```diff
@@ -39,82 +39,100 @@ describe(FilesService.name, () => {
+					expect(repo.findByPermissionRefIdOrCreatorId).toBeCalledWith(userId);
 					expect(result).toEqual([]);
 				});
 			});
 
-			describe('does have an access to some files', () => {
+			describe('does have an access or is creator of some files', () => {
 				it('should return an array containing proper file entities', async () => {
-					const { userId, accessibleFiles } = setup();
+					const { userId, accessibleOrCreatedFiles } = setup();
 
-					repo.findByPermissionRefId.mockResolvedValueOnce(accessibleFiles);
+					repo.findByPermissionRefIdOrCreatorId.mockResolvedValueOnce(accessibleOrCreatedFiles);
 
-					const result = await service.findFilesAccessibleByUser(userId);
+					const result = await service.findFilesAccessibleOrCreatedByUser(userId);
 
-					expect(repo.findByPermissionRefId).toBeCalledWith(userId);
-					expect(result).toEqual(accessibleFiles);
+					expect(repo.findByPermissionRefIdOrCreatorId).toBeCalledWith(userId);
+					expect(result).toEqual(accessibleOrCreatedFiles);
 				});
 			});
 		});
 	});
 
-	describe('removeUserPermissionsToAnyFiles', () => {
-		it('should not modify any files if there are none that user has permission to access', async () => {
+	describe('removeUserPermissionsOrCreatorReferenceToAnyFiles', () => {
+		it('should not modify any files if there are none that user has permission to access or is creator', async () => {
 			const userId = new ObjectId().toHexString();
-			repo.findByPermissionRefId.mockResolvedValueOnce([]);
+			repo.findByPermissionRefIdOrCreatorId.mockResolvedValueOnce([]);
 
-			const result = await service.removeUserPermissionsToAnyFiles(userId);
+			const result = await service.removeUserPermissionsOrCreatorReferenceToAnyFiles(userId);
 
 			expect(result).toEqual(0);
 
-			expect(repo.findByPermissionRefId).toBeCalledWith(userId);
+			expect(repo.findByPermissionRefIdOrCreatorId).toBeCalledWith(userId);
 			expect(repo.save).not.toBeCalled();
 		});
 
-		describe('should properly remove user permissions', () => {
-			it('in case of just a single file accessible by given user', async () => {
+		describe('should properly remove user permissions, creatorId reference', () => {
+			it('in case of just a single file (permission) accessible by given user and couple of files created', async () => {
 				const userId = new ObjectId().toHexString();
 				const userPermission = filePermissionEntityFactory.build({ refId: userId });
-				const entity = fileEntityFactory.buildWithId({ permissions: [userPermission] });
+				const entity = fileEntityFactory.buildWithId({ permissions: [userPermission], creatorId: userId });
+				const entity2 = fileEntityFactory.buildWithId({ creatorId: userId });
+				const entity3 = fileEntityFactory.buildWithId({ permissions: [userPermission] });
 
-				repo.findByPermissionRefId.mockResolvedValueOnce([entity]);
+				repo.findByPermissionRefIdOrCreatorId.mockResolvedValueOnce([entity, entity2, entity3]);
```

Comment:
Please move it to the `setup()` function.

Reply:
done